mediaqueries-5/Overview.bs

```diff
@@ -148,6 +148,59 @@ Detecting the ambient light level: the 'light-level' feature</h3>
 		</pre>
 	</div>
 
+<h3 id="environment-blending">
+Detecting the display technology: the 'environment-blending' feature</h3>
+
+    <pre class='descdef mq'>
+    Name: environment-blending
+    Value: opaque | additive | subtractive
+    For: @media
+    Type: discrete
+    </pre>
+
+    The 'environment-blending' media feature is used to query the characteristics of the user's display
+    so the author can adjust the style of the document.
+    An author might choose to adjust the visuals and/or layout of the page depending on the display
+    technology to increase the appeal or improve legibility.
+
+    The following values are valid:
+
+    <dl dfn-type=value dfn-for="@media/environment-blending">
+        <dt><dfn>opaque</dfn>
+        <dd>
+            The display is a traditional screen or monitor where there is a completely opaque back panel.
```

Comment:
Also: paper.

Suggestion: The document is rendered on an opaque medium, such as a traditional monitor or paper.

Reply:
@frivoal done!